### PR TITLE
Corrected spelling to align with rest of document

### PR DESCRIPTION
**Pull request by raginjason**

### CODE REVIEW

File: website/docs/guides/best-practices/how-we-structure/1-guide-overview.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Below is the complete file tree of the project we’ll be working through. Don
 ```markdown
 jaffle_shop
 ├── README.md
-├── analyses
+├── analysis
```
Comment (mirnawong1) [inline website/docs/guides/best-practices/how-we-structure/1-guide-overview.md]:
```suggestion
├── analyses
```

**Comment by mirnawong1** [comment]
hey @raginjason thanks so much for opening this pr! this prompted a great convo with @dave-connors-3 and it looks like `analyses` is the correct default setting. the rest of the page and section is actually incorrectly using `analysis` so I've gone ahead and made those corrections in your branch. I'll go ahead and merge this as well!

really appreciate your first contribution, it sparked some wonderful conversations! looking forward to your next one ✨

**Review by mirnawong1 — APPROVED** [review]
🚢

**Comment by raginjason** [comment]
Ah yes, as I continued through the docs it confused me more. Late last night I came to the same conclusion you did. I'm glad you caught it!

**Pull request merged**